src/client/Sidebar.js

```diff
@@ -0,0 +1,62 @@
+import Col from "react-bootstrap/Col";
+import Nav from "react-bootstrap/Nav";
+import React from "react";
+import Row from "react-bootstrap/Row";
+import Tab from "react-bootstrap/Tab";
+import { useTranslation } from "react-i18next";
+
+function Sidebar() {
```

Comment:
Could we add a snapshot test for this component? In the past, we've used the file structure: `client/components/*component_name*/index.js`
`client/components/*component_name*/index.test.js`
`client/components/*component_name*/__snapshots__` (files are added to this folder automatically)

I thought that file structure worked well, but I'm definitely open to other ideas too!

Reply:
ah right, sure!

Reply:
added!